src/test/java/calculator/domain/DelimiterTest.java

```diff
@@ -0,0 +1,41 @@
+package calculator.domain;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class DelimiterTest {
+
+  @Test
+  @DisplayName("Test split with default delimiter.")
+  public void testDefaultDelimiter() {
+    Delimiter defaultDelimiter = Delimiter.from(null);
+
+    String[] splitWithComma = defaultDelimiter.split("123,4");
+    String[] splitWithColon = defaultDelimiter.split("123:4");
+
+    assertArrayEquals(splitWithComma, new String[]{"123", "4"});
+    assertArrayEquals(splitWithColon, new String[]{"123", "4"});
```

Comment:
assertArrayEquals는 first parameter가 기대값이고 second parameter가 실제결과값입니다 ㅎㅎ 
```suggestion
    assertArrayEquals(new String[]{"123", "4"}, splitWithComma);
    assertArrayEquals(new String[]{"123", "4"}, splitWithColon);
```